src/util/regex.js

```diff
@@ -13,9 +13,11 @@
  * limitations under the License.
```

Comment:
These are a bit scary

Reply:
Eslint suggested these changes. They look safe to me.